### PR TITLE
Replace hardcoded card pages with isaaccarddecks

### PR DESCRIPTION
**Pull request by mwtrew**

### CODE REVIEW

File: src/app/components/content/IsaacCardDeck.tsx [hunk 1]
```diff
@@ -9,7 +10,15 @@ interface IsaacCardDeckProps {
...
-    return <div className={"card-deck isaac-cards-body " + classNameString}>
-        {doc?.cards?.map((props, i) => <IsaacCard key={i} doc={props}/>)}
-    </div>
+    return <Container>
+        {doc.title && <Row className="my-4">
+            <Col>
+                <h3 className="h-title text-center">{doc.title}</h3>
+            </Col>
+        </Row>}
+        <Row className={"card-deck isaac-cards-body my-3" + classNameString}>
```
Comment (chrisjpurdy) [inline src/app/components/content/IsaacCardDeck.tsx]:
```suggestion
        <CardDeck className={"card-deck isaac-cards-body my-3" + classNameString}>
```
Changing this `Row` to a `CardDeck` makes the cards stack nicely on small screen sizes. Currently both on live and here the cards look quite misaligned in this case.

File: src/app/components/content/IsaacCardDeck.tsx [hunk 2]
```diff
@@ -9,7 +10,15 @@ interface IsaacCardDeckProps {
...
+            {doc?.cards?.map((props, i) => <IsaacCard key={i} doc={props}/>)}
+        </Row>
```
Comment (chrisjpurdy) [inline src/app/components/content/IsaacCardDeck.tsx:21]:
```suggestion
        </CardDeck>
```

File: src/app/components/content/IsaacCardDeck.tsx [hunk 3]
```diff
@@ -9,7 +10,15 @@ interface IsaacCardDeckProps {
 
 export const IsaacCardDeck = ({doc, className}: IsaacCardDeckProps) => {
     const classNameString: string = className !== undefined ? className : "";
```
Comment (chrisjpurdy) [inline src/app/components/content/IsaacCardDeck.tsx]: This old `classNameString` stuff is a bit messy, so I've pushed a small commit that neatens the logic a little.

**Review by chrisjpurdy — CHANGES_REQUESTED** [review]
Looks good apart from a couple of visual things!

File: src/app/components/content/IsaacCardDeck.tsx (hunk 1 above)
Comment (chrisjpurdy) [reply src/app/components/content/IsaacCardDeck.tsx]: Best place to observe this is `/alevel`

Comment (mwtrew) [reply src/app/components/content/IsaacCardDeck.tsx]: Agreed, that looks much better.

File: src/app/components/content/IsaacCardDeck.tsx [hunk 4]
```diff
@@ -9,7 +10,15 @@ interface IsaacCardDeckProps {
...
-    return <div className={"card-deck isaac-cards-body " + classNameString}>
-        {doc?.cards?.map((props, i) => <IsaacCard key={i} doc={props}/>)}
-    </div>
+    return <Container>
```
[the remark this answers is not in the document]
Comment (mwtrew) [reply src/app/components/content/IsaacCardDeck.tsx]: Good spot, definitely cleaner-looking that way.

**Review by chrisjpurdy — APPROVED** [review]
Cool, LGTM! The text under the header on `/alevel` for example is a bit larger on live but I don't think that matters.

**Pull request merged**